docs/atomic-cluster.1.md

```diff
@@ -0,0 +1,52 @@
+% ATOMIC(1) Atomic Man Pages
+% Matthew Barnes
+% May 2016
+# NAME
+atomic-cluster - Manage cluster hosts
+
+# SYNOPSIS
+**atomic cluster [OPTIONS] SUBCOMMAND**
+
+**atomic cluster** will issue commands to a Commissaire server over HTTP.
```

Comment:
I'm not sure where manpages get packaged for AH these days but my thought was if we could temporarily delete these two pages in a spec file or whatever, that seems cleaner to me than a disclaimer.  Failing that, yeah I can add a line.